Task2/Program.cs

```diff
@@ -9,13 +9,8 @@ internal class Program
 
         static void Main(string[] args)
         {
-            foreach (var x in args)
-            {
-                foreach (var result in FindSubsequences(x))
-                    Console.Write($"{result} ");
-
-                Console.WriteLine();
-            }
+            foreach (var arg in args)
+                Console.WriteLine(string.Join(", ", FindSubsequences(arg)));
```

Comment:
The Join method was used for readability purposes. Will change if performance is critical.